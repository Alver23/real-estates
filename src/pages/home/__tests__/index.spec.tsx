// Dependencies
import { shallow, ShallowWrapper } from 'enzyme';

// Under test file
import HomePage from '../index';

describe('<HomePage />', () => {
  let component: ShallowWrapper;
  const props = {};

  beforeEach(() => {
    component = shallow(<HomePage {...props} />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });
});
