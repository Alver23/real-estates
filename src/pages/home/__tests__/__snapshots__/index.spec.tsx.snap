// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should save a snapshot of the component 1`] = `
<LayoutApp>
  <div
    className="m-auto container pt-14"
  >
    <h1
      className="font-semibold text-center text-primary text-2xl md:text-3xl"
    >
      Listas de favoritos
    </h1>
    <FavoritesList />
  </div>
</LayoutApp>
`;
